### PR TITLE
Add Ellipsis Loader Component

### PR DESCRIPTION
**Pull request by musangowope**
## Proposed changes
- Adds ElipsisLoader component and story for components
- Adds 2 test for ellipsis component (1. Testing the color the color of the ellipsis dots and 2. testing that the component had four )
...

## Types of changes

What type of changes does your code introduce?
_Put an `x` in the boxes that apply, leave blank if none_

- [x] Frontend
- [ ] Backend
- [ ] Bugfix
- [ ] New feature
- [ ] Component of Major Release
- [ ] Project Management
- [ ] Quality of Life

## Checklist

_Put an `x` in the boxes that apply._

- [ ] Local Cypress Tests Pass with my changes (`yarn cypress:run`)
- [ ] Lint and unit tests pass locally with my changes (`yarn lint && ./vendor/bin/phpunit`)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- ...

## Relevant Issues
https://github.com/bonlineza/objectives-and-key-results/issues/17

### CODE REVIEW

**Comment by musangowope** [comment]
> Changes to package.json require changes to lock files to be committed as well, once the lockfile gets added I can approve

it makes no change to `yarn.lock` file

File: src/components/EllipsisLoader/index.test.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    const wrapper = shallow(<EllipsisLoader dotColor="#666" />);
+    const dotElements = wrapper.find('.loader__lds-ellipsis__dot');
+
+    dotElements.forEach(dot => {
+      const dotBackgroundColor = dot.props().style.backgroundColor;
+      expect(dotBackgroundColor).toBe('#666');
+    });
+  });
+
+  it('test that for dots render', () => {
```
Comment (sscholle) [inline src/components/EllipsisLoader/index.test.js]: `four` not `for`

File: src/components/EllipsisLoader/index.test.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import Enzyme, { shallow } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
```
Comment (sscholle) [inline src/components/EllipsisLoader/index.test.js]: please use standard test setup file: `import 'test-util/setup';`

no need to configure the Adapter each time: see line 3 and 6 - replace with the above line

File: src/components/EllipsisLoader/index.test.js (hunk 1 above)
Comment (musangowope) [reply src/components/EllipsisLoader/index.test.js]: amended

File: src/components/EllipsisLoader/index.test.js (hunk 2 above)
Comment (musangowope) [reply src/components/EllipsisLoader/index.test.js]: amended

**Pull request merged**